src/client/ui/cursor/iframe-cursor.js

```diff
@@ -0,0 +1,39 @@
+import hammerhead from './../deps/hammerhead';
+import CURSOR_UI_MESSAGES from './messages';
+import { sendRequestToFrame } from '../../runner/utils/iframe';
+
+var browserUtils = hammerhead.utils.browser;
+
+//HACK: we can't get element under cursor in cross-domain iframe in IE without hide cursor.
+//Instead make 'getElement' method asynchronous, we set cursor to position by one pixel farther.
```

Comment:
In most browsers, the iframe's getElementFromPoint function ignores elements from the parent frame. But in IE it doesn't, and our cursor overlaps the target element. So, we move the cursor to a position one pixel farther to avoid this.